SConstruct

```diff
@@ -182,6 +183,13 @@ elif platform == "win32":
 		env.Install(build_dir, source)
 elif platform == "posix":
 	env.Append(CXXFLAGS=["-std=c++14","-include","global.hpp"])
+	env.Append(LIBPATH=Split("""
```

Comment:
Please remove the excessive indentation here, so that the closing `"""` is at the same indentation level as `env.Append`.